cmd/mount.go

```diff
@@ -67,13 +67,40 @@ func reload(fs *g8ufs.G8ufs, cmd *Cmd) error {
 }
 
 func mount(cmd *Cmd, target string) error {
-	fs, err := start(cmd, target)
+	if cmd.LogPath == "" {
+		cmd.LogPath = "/var/log/g8ufs.log"
+	}
+	cntxt := &daemon.Context{
+		PidFileName: cmd.PidPath,
+		PidFilePerm: 0644,
+		LogFileName: cmd.LogPath,
+		LogFilePerm: 0640,
+		WorkDir:     "./",
+		Umask:       027,
+	}
+
+	var (
+		fs  *g8ufs.G8ufs
+		err error
+	)
+
+	if cmd.Daemon {
```

Comment:
Is it possible that u do the mount first, before you daemonize then the child can maintain the mount while the parent exits. this way u won't have to wait in the flist module for the mount to happen.

Reply:
No it doesn't work I tried that, but it seems some file descriptor or something are not properly passed to the children, I'm thinking this problems comes from fuse or something, but I didn't managed to do it like that